tools/clang/unittests/HLSLExec/ExecutionTest.cpp

```diff
@@ -758,6 +758,12 @@ class ExecutionTest {
       return nullptr;
     }
 
+    if (!DoesDeviceSupportWaveMatrix(pDevice)) {
+      LogCommentFmt(L"WaveMatrix not supported on this device.");
+      WEX::Logging::Log::Result(WEX::Logging::TestResults::Skipped);
+      return nullptr;
```

Comment:
Does the caller gracefully handle not getting a device, even if a skip was logged here? (e.g. exit test)

From my web view not easy for me to quickly check myself I don't think.

Reply:
Yes.